runtime/ts/slot-consumer.ts

```diff
@@ -24,7 +24,7 @@ export class SlotConsumer {
   // Contains `container` and other affordance specific rendering information
   // (eg for `dom`: model, template for dom renderer) by sub id. Key is `undefined` for singleton slot.
   private _renderingBySubId: Map<string, {container?: {}}> = new Map();
-  private _innerContainerBySlotName: {} = {};
+  private _innerContainerBySlotId: {} = {};
```

Comment:
could remove underscore at some time as rename? We should only underscore member variables when they are shadowed by a public getter and also modified internally.

Reply:
Done, thanks.